designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -170,49 +193,11 @@
 # transition time is 10% of a 4 GHz period.  Note that we have to
 # create a clock before the transition constraint is applied
 # (appears to be related to the fact that these are internal nets)
-set mon_nets [get_pins {{ \\
-    iacore/*del_out_pi* \\
-    iacore/*pi_out_meas* \\
-    iacore/*del_out_rep* \\
-    iacore/*inbuf_out_meas* \\
-    iacore/*pfd_inp_meas* \\
-    iacore/*pfd_inn_meas* \\
-}}]
-foreach x [get_object_name $mon_nets] {{
+foreach x [get_object_name $adbg_clk_pins] {{
     create_clock -name clk_mon_net_$x -period {0.25*time_scale} [get_pins $x]
     set_max_transition {0.025*time_scale} -clock_path [get_clocks clk_mon_net_$x]
 }}
 
-# Set transition time for clk_async
```

Comment:
Just wanted to check that this removal is intentional.  I remember hearing about some synthesis issues related to constraints, so maybe this had to be deleted because it was causing problems?